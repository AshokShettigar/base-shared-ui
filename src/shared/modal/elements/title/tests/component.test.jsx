import React from 'react';
import { shallow } from 'enzyme';

import ModalHeaderSection from '../component';

const defaultProps = {
    children: '',
    onDismiss: jest.fn()
};

it('should render component', () => {
    const wrapper = shallow(<ModalHeaderSection {...defaultProps} />);

    expect(wrapper).toMatchSnapshot();
});
