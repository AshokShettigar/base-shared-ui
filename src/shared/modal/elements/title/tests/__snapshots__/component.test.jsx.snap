// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
<div
  className="root"
  data-component="ModalTitleElement"
>
  <div
    className="title"
  />
  <Icon
    className="icon"
    icon="close"
    onClick={[Function]}
    size="extra-small"
  />
</div>
`;
